topics/localization/index.md

```diff
@@ -0,0 +1,8 @@
+---
+aliases: localization, localisation
+display_name: Localisation
```

Comment:
Super minor detail: the directory name here is US-centric "locali**z**ation", but the display name, topic, and description have the UK-centric "locali**s**ation". Should we make them consistent? 

I probably have a slight preference for the 💤 , fwiw.

Reply:
Ooooo looks like [CI caught this](https://travis-ci.org/github/explore/builds/391481676?utm_source=github_status&utm_medium=notification):

```
  8) Failure:
topics::localisation topic#test_0003_does not add an alias that's already in use [/home/travis/build/github/explore/test/topics_test.rb:32]:
localization,  localisation are already aliased to localization, please remove from either 'localisation' or 'localization'.
Expected ["localization", " localisation"] to be empty.
```

There are a bunch of other failures 🤔 . Mind taking a look?

Reply:
solved in my new PR -- coming soon